docs/UserGuide.md

```diff
@@ -53,34 +53,58 @@ command to run the application.<br>
 
 ### Adding a client: `add`
 
-Adds a client to the Realodex. Note that Name, Address, Income, Phone Number, Family Size, Email and Tag are compulsory. You may input the parameters in any order (e.g. if 
-the command specifies `n/NAME a/ADDRESS`, `a/ADDRESS n/NAME` is also acceptable). A client can be a buyer, seller or both. This should be reflected by tagging `t/buyer` and/or `t/seller`.
+Adds a client to the Realodex. 
 
-Format: `add n/NAME p/PHONE i/INCOME e/EMAIL a/ADDRESS f/FAMILY t/TAG ...`
+Format: `add n/NAME p/PHONE i/INCOME e/EMAIL a/ADDRESS f/FAMILY t/TAG [r/REMARK]`
+
+- Note that `REMARK` is optional
+- You may input the parameters in any order (e.g. if
+the command specifies `n/NAME a/ADDRESS`, `a/ADDRESS n/NAME` is also acceptable). 
+- Note that the tag is to indicate if a client is a
+Buyer, Seller or both, so tags only accept "buyer" or "seller" as the input (case-insensitive).
 
 Examples:
-* `add n/John Doe p/98765432 i/20000 e/johnd@example.com a/311, Clementi Ave 2, #02-25 f/4 t/buyer`
-* `add n/Betsy Crowe a/Newgate Prison i/$0 f/1 p/94859694 e/betsyc@rocketmail.com t/seller`
+* `add n/John Doe p/98765432 i/20000 e/johnd@example.com a/311, Clementi Ave 2, #02-25 f/4 t/Buyer r/Owes $1000.`
+* `add n/Betsy Crowe a/Newgate Prison i/$0 f/1 p/94859694 e/betsyc@rocketmail.com t/Seller`
 
 ### Deleting a client : `delete`
 
-Deletes the specified client from the Realodex.
+Deletes the specified client from the Realodex. There are 2 ways to do so: 
```

Comment:
There is only one way to do so. Delete by name is not implemented (from our previous discussion, i recall that delete by name is not implemented for consistency with edit that only allows edit by index)